wave_matching/include/wave/matching/impl/ground_segmentation.hpp

```diff
@@ -47,8 +47,7 @@ void GroundSegmentation<PointT>::genPolarBinGrid() {
 
         if (sqrt(px * px + py * py + pz * pz) < this->params.rmax) {
             double ph = (atan2(py, px)) * (180 / M_PI);  // in degrees
-            if (ph < 0)
-                ph = 360.0 + ph;
+            ph = wave::wrapTo360(ph);
```

Comment:
This is already in `wave` namespace right?

Reply:
yep, I'll take it out